src/pulse3D/plate_recording.py

```diff
@@ -311,8 +312,8 @@ class PlateRecording:
     def __init__(
         self,
         path,
-        calc_time_force=True,
```

Comment:
What was the purpose of this arg? It looks like `force_df` is replacing it, so just curious

Reply:
It was to prevent PlateRecording from running `_process_plate_data` and speed up analysis time by using existing data from a parquet file stored in s3 that gets written on initial run. Thats still happening, but the using the `from_dataframe` method now.